Makefile

```diff
@@ -1,23 +1,37 @@
 export SERVICE_NAME=xds-relay
+export GOREPO=${GOPATH}/src/github.com/envoyproxy/${SERVICE_NAME}
+SOURCE_FILES?=$$(go list ./... | grep -v integration)
```

Comment:
I couldn't find a simpler way of skipping the integration test I'm adding in this PR. Maybe there's a different way of structuring files/packages in such a way that this is not necessary. Feedback on how to do this is more than welcome.